internal/install/recipe_installer.go

```diff
@@ -718,17 +716,24 @@ func (i *RecipeInstall) finishHandlingFailure(recipeName string) {
 	}
 }
 
-func checkNetwork(nrClient *newrelic.NewRelic) {
+func checkNetwork(nrClient *newrelic.NewRelic) error {
 	err := nrClient.TestEndpoints()
 	if err != nil {
 
 		proxyConfig := httpproxy.FromEnvironment()
 
 		log.Debugf("proxyConfig: %+v", proxyConfig)
 		if proxyConfig.HTTPProxy != "" || proxyConfig.HTTPSProxy != "" || proxyConfig.NoProxy != "" {
-			log.Warn("Proxy settings have been configured but we are still unable to connect to the New Relic platform.  You may need to adjust your proxy environment variables.  https://github.com/newrelic/newrelic-cli/blob/main/docs/GETTING_STARTED.md#using-an-http-proxy")
+			log.Warn("Proxy settings have been configured, but we are still unable to connect to the New Relic platform.")
```

Comment:
It might be helpful to output the current proxy config here so it's easy to check if they have the correct settting?